webapp/src/store/mergeStoreData.spec.ts

```diff
@@ -125,6 +125,25 @@ describe('mergeStoreData()', () => {
     const result = mergeStoreData(inMemory, fromRepo);
     expect(result).toEqual(fromRepo);
   });
+
+  it('should not return undefined if both memory and repo are falsy (undefined)', () => {
+    const inMemory: StoreData = {
+      languages: {
+        sv: {},
+      },
+      messages: [],
+    };
+
+    const fromRepo: StoreData = {
+      languages: {
+        sv: {},
+      },
+      messages: [mockMessage('any.message.id', 'Default message')],
+    };
+
+    const result = mergeStoreData(inMemory, fromRepo);
+    expect(result).toEqual(fromRepo);
```

Comment:
`toEqual` is not strict enough to catch this bug. It fins no difference between `{'any.message.id':undefined}` and `{}`.

Reply:
fixed by use toStrictEqual